rotkehlchen/globaldb/cache.py

```diff
@@ -140,21 +134,22 @@ def globaldb_get_general_cache_last_queried_ts_by_key(
     return Timestamp(result[0])
 
 
-def globaldb_delete_general_cache(
+def globaldb_delete_cache(
```

Comment:
I don't think this makes much sense. I think we need different deletion functions.

For the general cache deletion makes sense.
For unique cache maybe we need to delete at some point, but in every place I can imagine we need to replace.
So we need a different function/use for it. And then this if can be removed from here.